nvflare/fuel/utils/gpu_utils.py

```diff
@@ -16,25 +16,53 @@
 from typing import List
 
 
-def get_host_gpu_ids() -> List:
+def use_nvidia_smi(
+    query: str,
+    report_format: str = "csv"
+):
     try:
         process = subprocess.Popen(
```

Comment:
subprocess.run() may be better suited this case.

Reply:
Thanks fixed